src/common/components/clusterDetail/OperatorsProgressItem.tsx

```diff
@@ -72,25 +74,32 @@ export function getOperatorsIcon(status: OperatorStatus | 'pending') {
 
 type OperatorListProps = {
   operators: MonitoredOperatorsList;
+  openshiftVersion: Cluster['openshiftVersion'];
 };
 
 type OperatorsPopoverProps = OperatorListProps & {
   children: React.ComponentProps<typeof Popover>['children'];
 };
 
-const OperatorsPopover = ({ operators, children }: OperatorsPopoverProps) => {
+const OperatorsPopover = ({ operators, openshiftVersion, children }: OperatorsPopoverProps) => {
   const { t } = useTranslation();
+  const featureSupportLevel = useFeatureSupportLevel();
+
   return (
     <Popover
       headerContent={<div>{t('ai:Operators')}</div>}
       bodyContent={
         <List className="operators-progress-item__operators-list">
-          {operators.map((operator) => {
+          {operators.map((operator: MonitoredOperator) => {
             let status = operator.status ?? 'pending';
             if (operator.status === 'available') {
               status = 'installed';
             }
-            const operatorName = operator.name as OperatorName;
+            const operatorName = getActualLVMOperatorName({
```

Comment:
I see this fixes the problem in the cluster installation page, however there are other places where we are using `operatorLabels` and we may not have adapted the operator name.

So I'm thinking that it would be better to move the logic to adapt the operator name to the `operatorLabels`, and not have to remember to call the `getActualLVMOperatorName` everytime we want the operator name.

Reply:
Good idea. :+1: